<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scoreWindow</class>
 <widget class="QMainWindow" name="scoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="score_frame1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="you_typed_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>You typed (x) words in (y) seconds and your score was (z) w.p.m.!</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="score_frame2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>410</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="outcome_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>No highscore was set. Better luck next time!</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="score_frame3">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>70</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ok_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="statusTip">
      <string>Brings you back to the main menu</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
